samples/NameParsing.js

```diff
@@ -0,0 +1,58 @@
+enyo.kind({
+    name: "ilib.sample.NameParsing",
+    kind: "FittableRows",
+    classes: "onyx ilib-onyx-sample enyo-fit",
+    
+    components: [
+        {kind: "Scroller", fit: false, components: [
+            {kind: "FittableColumns", components: [
+                /* Header with selecting locale */
+                {kind: "ilib.sample.ChooseLocale", name: "localeSelector", onSelectedLocale: "setLocale"},
```

Comment:
I think we can remove `onSelectedLocale: "setLocale"` as this is already defined in `ilib.sample.ChooseLocale`.